### DISCUSSION TITLE
What is the deprecation policy of github runner releases?

### DISCUSSION THREAD

**Discussion by hania-elayoubi** [actions/runner#1992] [General] [answered]
[v2.291.1](https://github.com/actions/runner/releases/tag/v2.291.1), released on April 29 has now (July 11) been deprecated:

```
An error occurred: Runner version v2.291.1 is deprecated and cannot receive messages.
```

This deprecation seems unusually fast. 

If the releases become deprecated in under 3 months, what is the policy of backward compatibility so we never run with an old release in the future?

**Comment by TingluoHuang** [comment, marked as answer]
https://docs.github.com/en/actions/hosting-your-own-runners/autoscaling-with-self-hosted-runners#controlling-runner-software-updates-on-self-hosted-runners

**Reply by tjsousa** [reply]
This documentation doesn't seem available anymore. what should we expect in terms of deprecation policy.
